### ISSUE TITLE
Arduino preprocessing stripping comments when creating .cpp file

### ISSUE THREAD

**Issue by WestfW** [arduino/Arduino#3598]
Why has the IDE started stripping comments when preprocessing .ino files to .cpp files in the build directory?  This is liable to make disassembly and real debugging more difficult (for those few who do it), and violates the principal of doing as little pre-processing as possible, and I don't see any advantage to doing it...

**Comment by Chris--A** [comment]
Yup very bad, it doesn't actually do it properly:

```arduino



//This is a valid comment. \
so is this, but not today!

void setup() {}
void loop() {}
```

http://stackoverflow.com/questions/7059549/c-multi-line-comments-using-backslash

**Comment by ffissore** [comment]
Sorry, this issue belongs to a long list of issue tagged with https://github.com/arduino/Arduino/labels/Component%3A%20Preprocessor

**Comment by Chris--A** [comment]
Ah, I had forgotten I had a PR for this too: https://github.com/arduino/Arduino/pull/3240

And this is a duplicate of #956

**Comment by ffissore** [comment]
Fixed by #3779
